system-tests/docker-compose.yml

```
@@ -171,6 +171,7 @@ services:
       context: ./resources/cli-tools
     volumes:
       - ./resources:/resources
+    tty: true
```

```suggestion```
Should be removed unless there is a reason for the interactive terminal

---

Already addressed in Marc's comment. Has been removed.